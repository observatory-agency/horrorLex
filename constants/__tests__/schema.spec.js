const schema = require('../schema');

describe('schema', () => {
  it('should match the snapshot', () => {
    expect(schema).toMatchSnapshot();
  });
});
