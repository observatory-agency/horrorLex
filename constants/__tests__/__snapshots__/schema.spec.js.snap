// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`schema should match the snapshot 1`] = `
Object {
  "book": Object {
    "validator": Object {
      "$jsonSchema": Object {
        "additionalProperties": false,
        "bsonType": "object",
        "properties": Object {
          "_id": Object {
            "bsonType": "objectId",
          },
          "name": Object {
            "bsonType": "string",
            "description": "must be a string and is required",
          },
          "tags": Object {
            "bsonType": "array",
            "items": Object {
              "bsonType": "string",
              "description": "required and must be a string",
            },
            "minItems": 1,
          },
        },
        "required": Array [
          "name",
          "tags",
        ],
      },
    },
  },
}
`;
